movie.rb

```diff
@@ -0,0 +1,10 @@
+require_relative './item'
+
+class Movie < Item
+  attr_reader true
+
+  def initialize
+    super(true)
+    @silent = true
```

Comment:
I suggest you assign the initial value "true" and the silent variable as arguments to the constructor.
Like this: def initialize(silent: true)